lib/chef/provider/package/windows/exe.rb

```diff
@@ -0,0 +1,159 @@
+#
+# Author:: Matt Wrock <matt@mattwrock.com>
+# Copyright:: Copyright (c) 2015 Chef Software, Inc.
```

Comment:
Given this is mostly copy-paste from the original provider, you add another author line for `Author:: Seth Chisamore <schisamo@chef.io>` and add 2011 to the copyright?